src/logging.cpp

```diff
@@ -184,6 +184,93 @@ bool GetLogCategory(BCLog::LogFlags& flag, const std::string& str)
     return false;
 }
 
+std::string LogLevelToStr(const BCLog::Level& level)
+{
+    switch (level) {
+    case BCLog::Level::None:
+        return "none";
+    case BCLog::Level::Debug:
+        return "debug";
+    case BCLog::Level::Info:
+        return "info";
+    case BCLog::Level::Warning:
+        return "warning";
+    case BCLog::Level::Error:
+        return "error";
+    default:
```

Comment:
the default case will disable the compiler warning (see dev notes)

Reply:
Thanks for catching it! Replaced the default case with an `assert(false)`